### ISSUE TITLE
jupyter-notebook server won't receive "ctrl+c" interrupt

### ISSUE THREAD

**Issue by slee981** [holzschu/a-shell#167]
First off - great app! Very happy to find it. 

I ran `$ jupyter-notebook` and was able to get the localhost server to launch as expected. However when I try to interrupt and stop the server it won't receive any input. 

Any other reports of this? 

**Environment**: iOS 14.3, iPad 7th Generation

**Comment by holzschu** [comment]
Yes, I have observed the same issue. I have no idea how to force termination of the jupyter-notebook server once it's started, since it is not listening to standard input. Force-terminating the app by sliding the window upwards works for me. 

It is not considered a great priority because there is another app that offers jupyter notebooks specifically (Carnets), and because there are a lot more limitations to running jupyter notebooks in a-Shell (cannot run python inside notebooks, cannot create files, cannot access files from other apps...) But if you have application scenarios where this is really problematic, I can change the priority.

**Comment by slee981** [comment]
Ah, that makes sense - no reason on my end to increase priority. Just wanted to point it out if it was an unknown issue. Thanks!
